### PR TITLE
[TASK] Reorganizing the Settings page (RT-3299)

### PR DESCRIPTION
**Pull request by Mesrop**
Conflicts:
	src/jade/tabs/advanced.jade

### CODE REVIEW

**Comment by Mesrop** [comment]
@vhpoet Please, review this in details before merging/
Thanks!

File: src/jade/tabs/gatewaysettings.jade [hunk 1]
```diff
@@ -0,0 +1,94 @@
+section.col-xs-12.content(ng-controller="GatewayCtrl")
+  div(ng-show='debug') This page is not available in debug mode
```
Comment (jublo) [inline src/jade/tabs/gatewaysettings.jade]: This needs to be localizable.

File: src/jade/tabs/gatewaysettings.jade [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+                .col-xs-3.col-sm-3.col-md-2
+                  button.btn.btn-block.btn-danger.btn-xs.submit#delete(type="button", ng-click='deleteUrl("'+urlType+'")', l10n) Delete
+              .col-xs-3.col-sm-3.col-md-1.text-center#cancel
+                a.btn.btn-cancel.btn-block(href="", ng-click='cancelEdit("'+urlType+'")', l10n) cancel
+        +urlForm('bridge', 'bridgeForm', 'Bitcoin bridge', {url: '{{options.bridge.out.bitcoin}}', ngModel: 'options.bridge.out.bitcoin'}, 'btcDomain', true)
+        form.trust-line-form(name="accountsAdvForm", ng-submit='save("advanced_feature_switch")')
+          .descriptor(l10n) Trust line
+          .row.advanced-form-row(ng-hide="edit.advanced_feature_switch")
+            .col-xs-6.col-sm-5.col-md-3
+              .description Advanced settings
```
Comment (jublo) [inline src/jade/tabs/gatewaysettings.jade]: This needs to be localizable, too.

File: src/jade/tabs/gatewaysettings.jade [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+            .col-xs-3.col-sm-4.col-md-4
+              .description(ng-show="options.advanced_feature_switch", l10n) Show
+              .description(ng-hide="options.advanced_feature_switch", l10n) Hide
+            .col-xs-3.col-sm-3.col-md-2
+              a.btn.btn-cancel.btn-block(href="", ng-click="edit.advanced_feature_switch = true", l10n) edit
+          .row.row-padding-small.edit-trust-line(ng-show="edit.advanced_feature_switch")
+            .col-xs-6.col-sm-5.col-md-3
+              .description Advanced settings
```
Comment (jublo) [inline src/jade/tabs/gatewaysettings.jade]: Same here.

**Comment by Mesrop** [comment]
@jublo done

**Comment by vhpoet** [comment]
@Mesrop the controller and file names should be the same

**Comment by Mesrop** [comment]
@vhpoet Done.

**Comment by Mesrop** [comment]
@vhpoet pls check

**Comment by vhpoet** [comment]
@Mesrop gatewaysettings.jade and gatewaysetting.controller.js? setting or settings?

**Pull request closed without merging**